src/margin/config.json

```
@@ -0,0 +1,293 @@
+        "decimals": 6
+      },
+      {
+        "symbol": "ETH",
+        "mint": "AdkhsMLZo9RCfg4ZvFsp8gXsg3F1dmPxmw7FhX8WFeKW",
+        "decimals": 6
+      },
+      {
+        "symbol": "SOL",
+        "mint": "8sV64cK2dTmuUcLdQbQhuWp23eKXEWD9nrPSmw2AWeRC",
+        "decimals": 9
+      },
+      {
+        "symbol": "USDC",
+        "mint": "CgrwTWWUvchjiqAYoeqFdCwSyAwerQvehfgvQ3GpFJto",
+        "decimals": 6
+      }
+    ],
+    "oracles": [
+      {
+        "symbol": "BTC/USD",
+        "address": "8QzWALgDuBK45wnhx3fVXbLoQEoYNJbhpXPXhGJKRfUr"
+      },
+      {
+        "symbol": "ETH/USD",
+        "address": "6bWG7buAJDgwJBzGov6maCUmn8249RwKZPS1dPaeheEd"
+      },
+      {
+        "symbol": "SOL/USD",
+        "address": "EYoruynF7CxkCCK1DXVdfy96YjpFMSps5bxUnzf2K2qX"
+      }
+    ],
+    "markets": [
+      {
+        "symbol": "BTC/USDC",
+        "market": "BgrcTJptiZ2YMvfYuxEcRKChzgrLQH2NerkaDpfXYgPg",
+        "baseMint": "F4aQGmSARNyjFXJAYmNGue98dpsH1GawkSjgNZ8tcWpi",
+        "baseSymbol": "BTC",
+        "baseDecimals": 6,
+        "baseLotSize": 100,
+        "quoteMint": "CgrwTWWUvchjiqAYoeqFdCwSyAwerQvehfgvQ3GpFJto",
+        "quoteSymbol": "USDC",
+        "quoteDecimals": 6,
+        "quoteLotSize": 10,
+        "requestQueue": "L7ygGudBDqfBXM1zvsPxSsrYpY7yygXPrFdyR3gCapa",
+        "eventQueue": "FAyEusyt3neyg9D1jbUcyd6exHttc7sCwW9NhtipRnTm",
+        "bids": "FscwVXnfBasZJDGMsNxagJ87jaR4tcD1YoxwTFET47UC",
+        "asks": "9cFbYD94kBvAhPTqZU6fmHx5MBf1GoCskV9whgWF4DYx"
+      },
+      {
+        "symbol": "ETH/USDC",
+        "market": "8sGEZTZz5xewJq8vADEpzAUte1qpyLAxgUmKFDGSjCYL",
+        "baseMint": "AdkhsMLZo9RCfg4ZvFsp8gXsg3F1dmPxmw7FhX8WFeKW",
+        "baseSymbol": "ETH",
+        "baseDecimals": 6,
+        "baseLotSize": 1000,
+        "quoteMint": "CgrwTWWUvchjiqAYoeqFdCwSyAwerQvehfgvQ3GpFJto",
+        "quoteSymbol": "USDC",
+        "quoteDecimals": 6,
+        "quoteLotSize": 10,
+        "requestQueue": "7HTDMLwkA4BAacmNA7JpBrv1iWTXH8SkRNtgctXaze5s",
+        "eventQueue": "AKobYahe3cWJLd2Q9EV5EYqYXiLMuVFYZmCkRTTxJSoJ",
+        "bids": "9rPeUSCBDEe2B2w1CSPm5j7B7H1TFdcPbAPQTmtHcrmu",
+        "asks": "8PpCHLHAP8KDuuEfNgjffJKWjko2TgeX1Kd4Zchz1hbr"
+      },
+      {
+        "symbol": "SOL/USDC",
+        "market": "13pNBLHnbDFkpr6GZ5ZHP1XvsQQ4pz9NheX8i3ZHNGNp",
+        "baseMint": "8sV64cK2dTmuUcLdQbQhuWp23eKXEWD9nrPSmw2AWeRC",
+        "baseSymbol": "SOL",
+        "baseDecimals": 9,
+        "baseLotSize": 100000000,
+        "quoteMint": "CgrwTWWUvchjiqAYoeqFdCwSyAwerQvehfgvQ3GpFJto",
+        "quoteSymbol": "USDC",
+        "quoteDecimals": 6,
+        "quoteLotSize": 100,
+        "requestQueue": "DnDyBh8BsWse7yuusQSxF1L8qymgpAuUt4DULvbZzCJj",
+        "eventQueue": "EXGhdJnBWYteeoweNreQfPHUDPS6UeGwda22eia1d8Fo",
+        "bids": "8H2J9uMNbKhKVGPb48uBDLHChk37wxxJhPcYZNJ5C56U",
+        "asks": "C7bkVyYs5WDZYQsmjsrdViZK8G9JfF1CfKfSoYpw52La"
+      }
+    ]
+  },
+  "mainnet": {
```

```suggestion
  "mainnet-beta": {
```

Stick to the standard names for clusters